package.json

```diff
@@ -18,7 +18,8 @@
 		"vscode": "^1.61.0"
 	},
 	"extensionKind": [
-		"workspace"
+		"workspace",
+		"ui"
```

Comment:
Added this based on user requests.  Should help with running in various VSCode contexts, even if command-server might not work there